@nav-frontend/react-accordion/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@nav-frontend/react-accordion",
```

Comment:
Jeg er enig i at accordion nok ikke er det beste navnet. Hva med expansion panel? (https://material.io/archive/guidelines/components/expansion-panels.html)

Reply:
Renamet til `expansionpanel` for nå 👍